City-of-Helsinki/drupal-hdbt
# UHF-X Updated node modules.

# Node modules update

## What was done
* Tested all suggested node modules updates and checked that nothing was terminally changed.

## How to test
* `git checkout UHF-X_node_modules_update`
* `nvm use && npm i && npm run build`
* Check that none of the distributed styles have changed. 

## Designers review

One of the checkboxes below needs to be checked (`[x]`)
